pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [build-system]
 # Minimum requirements for the build system to execute.
 requires = [
-    "setuptools",
+    "setuptools>=38.3.0",  # version with most `setup.cfg` bugfixes
```

Comment:
Tiny suggestion: I recommend >=42 for use in pyproject.toml. I spend about an hour once debugging a broken build, and it turned out I didn't specify a minimum, and it was grabbing a cached copy of 41.somthing instead of the latest version, which had critical bugs for PEP 517 builds. If you don't make a PEP 517 build, then this is ignored anyway, and if you are building PEP 517, you want 42+ and can always get it. ;) See https://scikit-hep.org/developer/packaging#pep-517518-support-high-priority

PS: Great work on the NumPy requirements, that's what I came here to look at. :)

Reply:
It might have been 40.something, but whatever it was the devs told me to go with 42+.

Reply:
Thanks, we will make the change then!

Reply:
> See https://scikit-hep.org/developer/packaging#pep-517518-support-high-priority

Nice read!

> PS: Great work on the NumPy requirements, that's what I came here to look at. :)

Many thanks! I